### ISSUE TITLE
[Spec]  Chart view

### ISSUE THREAD

**Issue by krdmllr** [xamarin/XamarinCommunityToolkit#184]
# [The feature] 

[Microcharts](https://github.com/dotnet-ad/Microcharts) is a simple and well known chart library for xamarin and xamarin.forms. While it is simple, its dependency to SkiaSharp bring a big size overhead. By using the new Xamarin.Forms Shapes API, we could create a part of Microcharts to add simple Chart views to this library which would not have the SkiaSharp overhead.

The issue of using the Shapes API is that you have to nest Shapes to create multi-colored Charts and nest Labels to show descriptions since the Shapes API does not support layering.

# API

These are the charts we could port:

* Bar Chart
* Donut Chart
* Line Chart
* Pie Chart
* Point Chart
* Radar Chart
* Radial Gauge Chart
 
# Scenarios

Simple charts to show data.

# Platform Compatibility 

All platforms that are supported by Xamarin.Forms shapeshttps://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/shapes/

# Difficulty : medium

**Comment by jfversluis** [comment]
Yes! Awesome! As already pointed out in the chat, @jsuarezruiz was planning on something like this as well. So, please decide whoever wants to do this or even better; collaborate! :D
